### PR TITLE
Only set uiState cookies for logged in users

### PR DESCRIPTION
**Pull request by broskoski**

### CODE REVIEW

**Comment by dzucconi** [comment]
To me this seems like the wrong level to handle this. (It's fine I don't really care)

**Comment by broskoski** [comment]
Hm, where would you handle it?

**Comment by dzucconi** [comment]
At the level of the caller: like why does a logged out user even need a UIState?

**Comment by broskoski** [comment]
@dzucconi hows that sire

File: models/ui_state.coffee [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Backbone = require 'backbone'
 cookies = require 'cookies-js'
 { keys, each } = require 'underscore'
+CurrentUser = require './current_user.coffee'
```
Comment (dzucconi) [inline models/ui_state.coffee]: not needed

File: components/path/client/path_view.coffee [hunk 2]
```diff
@@ -17,7 +17,7 @@ module.exports = class PathView extends Backbone.View
     @setupSubViews()
 
   setupSubViews: ->
-    if (sd.USER or sd.CHANNEL) and sd.CURRENT_USER
+    if (sd.USER or sd.CHANNEL) and CurrentUser.orNull()
```
Comment (dzucconi) [inline components/path/client/path_view.coffee]: How's this any different?

**Pull request closed without merging**